source/fab/tools/tool_repository.py

```diff
@@ -72,6 +72,12 @@ def __init__(self):
                     Ar, Fcm, Git, Psyclone, Rsync, Subversion]:
             self.add_tool(cls())
 
+        # Add the common shells. While Fab itself does not need this,
+        # it is a very convenient tool for user configuration (e.g. to
+        # query nc-config etc)
+        for shell_name in ["bash", "sh", "ksh", "dash"]:
+            self.add_tool(Shell(shell_name))
```

Comment:
See previous comment about supporting non-standard shells.

Reply:
Order changed to have `sh` first.